libflux/go/libflux/buildinfo.gen.go

```diff
@@ -325,6 +325,8 @@ var sourceHashes = map[string]string{
 	"stdlib/planner/window_push_test.flux":                                                        "779f83caca79fec5e4129cb6d1b05b2fbcbb45b479acab69e075ad2ad62d9df2",
 	"stdlib/planner/window_sum_eval_test.flux":                                                    "460d45de75419da033d9bdf0b9a8c0d1bbdb57ed0d02f44d5f55d4d4b45086f1",
 	"stdlib/planner/window_sum_push_test.flux":                                                    "eb8a9bb757503a40da2f397acce7fda67cd71419f1eae420fb3c171d2f01d185",
+	"stdlib/polylinesimp/polylinesimp.flux":                                                       "14e941d1205a2ea89f4e01ea5c38ad4d70b35966745741397c4d02668d0ee279",
+	"stdlib/polylinesimp/polylinesimp_test.flux":                                                  "e1437db6cda53862420571117c90a346f45118feb82c202106a5204cde51e97d",
```

Comment:
Per the checklist:  
> If new packages are being introduced to stdlib, link to Working Group discussion notes and ensure it lands under experimental/

Should this go into `experimental/` to start in case the team wants to fuss with the public API?

Reply:
Thanks @onelson will move it to ### experimental/